CONTRIBUTING.md

```diff
@@ -55,7 +55,7 @@ again.
 
 [forking]: https://help.github.com/articles/fork-a-repo
 [golint]: https://github.com/golang/lint
-[golint readme]: https://github.com/golang/lint/blob/master/README
+[golint readme]: https://github.com/golang/lint/blob/master/README.md
```

Comment:
hmm, I'm not sure why we linked directly to the file in the first place.  How about https://github.com/golang/lint/#readme ?